### PR TITLE
Job manager client connector layer re-write

### PR DESCRIPTION
**Pull request by tbak**

### CODE REVIEW

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java [hunk 1]
```diff
@@ -16,12 +16,14 @@
 
 package com.netflix.titus.runtime.connector;
 
+import com.netflix.archaius.api.annotations.Configuration;
 import com.netflix.archaius.api.annotations.DefaultValue;
 
+@Configuration(prefix = "titus.grpcClientTunnables")
 public interface ChannelTunablesConfiguration {
 
     long DEFAULT_REQUEST_TIMEOUT_MS = 10_000;
```
Comment (corindwyer) [inline titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java]: can you change this to `60_000`?

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java [hunk 2]
```diff
@@ -16,12 +16,14 @@
 
 package com.netflix.titus.runtime.connector;
 
+import com.netflix.archaius.api.annotations.Configuration;
 import com.netflix.archaius.api.annotations.DefaultValue;
 
+@Configuration(prefix = "titus.grpcClientTunnables")
 public interface ChannelTunablesConfiguration {
```
Comment (corindwyer) [inline titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java]: Can you rename this file to `GrpcRequestConfiguration`?

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java (hunk 1 above)
Comment (tbak) [reply titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java]: Yes

File: titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java (hunk 2 above)
Comment (tbak) [reply titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/ChannelTunablesConfiguration.java]: Yes

**Pull request merged**